src/catalua.h

```diff
@@ -3,9 +3,67 @@
 #define CATALUA_H
 
 #include "int_id.h"
+#include "enums.h"
+#include "item.h"
 
 #include <string>
 #include <sstream>
+#include <vector>
+#include <list>
+
+#ifdef LUA
+extern "C" {
```

Comment:
Do you really need to drag in all these Lua headers? If it's just for the `lua_State *`, it should be defined in *one* of those headers, the other two are not needed. But Why include them here in the header anyway? Just remove the parameter from `Save` and use the global `lua_State *`, what many other functions in that file already do.

Reply:
I've moved these headers from `catalua.cpp` to `catalua.h`. Is it that bad?

Reply:
Thanks, that worked!